cdc/generation.cc

```diff
@@ -957,16 +963,8 @@ future<> generation_service::legacy_handle_cdc_generation(std::optional<cdc::gen
         co_return;
     }
 
-    if (!_sys_ks.local().bootstrap_complete() || !_sys_dist_ks.local_is_initialized()
-            || !_sys_dist_ks.local().started()) {
```

Comment:
Can we at least leave the `_sys_dist_ks` as sanity checks (`on_internal_error`)?

Reply:
OK, will keep one